cognito-pre-token-generation-openid/src/test/java/no/unit/nva/cognito/UserSelectionUponLoginHandlerTest.java

```diff
@@ -235,6 +236,19 @@ void shouldNotCreateUserForPersonsTopOrganizationWhenPersonHasNotLoggedInBeforeA
         assertThat(actualUsers, is(empty()));
     }
 
+    @Test
+    void shouldLogUserAttributesIfNinExtractFails() {
+        var personLoggingIn = scenarios.personWithoutNin();
+        var event = newLoginEvent(personLoggingIn.nin(), NON_FEIDE);
+        var testAppender = LogUtils.getTestingAppenderForRootLogger();
+
+        assertThrows(NoSuchElementException.class, () -> handler.handleRequest(event, context));
+
+        assertThat(testAppender.getMessages(), containsString("Could not extract required data from request"));
+        assertThat(testAppender.getMessages(), containsString(
+            "User name: null, userPoolId: null, input request: CognitoUserPoolPreTokenGenerationEvent.Request"));
+    }
```

Comment:
"CognitoUserPoolPreTokenGenerationEvent.Request" is not very useful in logs. Would be more interested in its content. "SOME" "VALUE"

Reply:
sorry, did not white the full line: it looks like this:
User name: null, userPoolId: null, input request: CognitoUserPoolPreTokenGenerationEvent.Request(super=CognitoUserPoolEvent.Request(userAttributes={SOME=VALUE}), clientMetadata=null, groupConfiguration=null)

where all data is included as far as I can see.

updating test to include all data